irksome/ButcherTableaux.py

```diff
@@ -17,14 +17,20 @@ class ButcherTableau(object):
     :arg c: a 1d array containing weights at which time-dependent
             terms are evaluated.
     :arg order: the (integer) formal order of accuracy of the method
+    :arg embedded_order: If present, the (integer) formal order of
+            accuracy of the embedded method
+    :arg gamma0: If present, the weight on the explicit term in the
+            embedded lower-order method
     """
-    def __init__(self, A, b, btilde, c, order):
+    def __init__(self, A, b, btilde, c, order, embedded_order, gamma0):
```

Comment:
It looks like this assumes that every `ButcherTableau` now has an embedded method.  Maybe we need an intermediate class like `EmbeddedButcherTableau`?

Reply:
I guess (seeing below) that you let `None` be an option for these.
Did you weigh subclassing versus checking for `None`?

Reply:
I didn't see a clean way to do this.  All of the schemes we currently have implemented are _essentially_ of type `CollocationButcherTableau` (because there's a general rule there, and we borrow from that case for LobattoIIIC).  So, we check that a `btilde` exists instead...  Happy to do this another way, but I don't currently see advantages or disadvantages to what is done now